kocify.sh

```diff
@@ -46,6 +54,19 @@ function ubermix_install_software {
   done
 }
 
+#https://raw.githubusercontent.com/kidsoncomputers/documentation/master/uganda/2019/install-packages.sh
+function install_software {
+  echo "================================="
+  echo "= Installing packages"
+  echo "================================="
+  software=( $(curl -sSL https://raw.githubusercontent.com/roninsphere/kocifier/mexico2019/packages.txt | sed '/^ *#/d;s/#.*//' ) )
```

Comment:
Is `packages.txt` in the `kidsoncomputers` repo? Or in `roninsphere` ?

Reply:
Yes, the packages.txt and the script kocify.sh are on my fork (roninsphere) at the moment; the urls will need to change back to kidsoncomputers when merged.